draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -940,19 +951,21 @@
    <xref target="cdn-pid">CDN Provider ID (PID)</xref> to detect and prevent loops as follows:
    <list style="symbols">
    <t>
-   uCDNs that originate a new trigger request SHOULD specify their CDN provider ID using 
+   uCDNs that originate a new trigger request SHOULD specify their CDN provider ID using the
    trigger "cdn-path" attribute (see <xref target="ci-trigger-resource"/> for details).
    </t>
    <t>
-   dCDN that receives a trigger creation request that contains "cdn-path" attribute MUST reject a 
-   trigger that already contains its own CDN PID in the CDN path.
+   dCDN that receives a trigger creation request that contains "cdn-path" attribute SHOULD check it for its own CDN PID.
+   If dCDN's PID is already present and dCDN is not the CDN initiating the trigger, this condition likely indicates a loop,
+   In such case, dCDN MUST reject the trigger, resulting in trigger being rejected back to its originating uCDN.
```

Comment:
This would read better if we said something like:
"...which would result in a trigger rejection being returned to the originating uCDN."